INSTALL.rst.in

```diff
@@ -175,3 +175,36 @@ Please see the detailed instructions for Windows in the file
 .. _jack for os x: http://www.jackosx.com/
 .. _pyliblo: http://das.nasophon.de/pyliblo/
 .. _pyyaml: https://pypi.python.org/pypi/PyYAML
+
+Bash on Windows
+---------------
+Using the latest insider preview of Windows adds a subsystem for Linux and users can use Bash just like a Linux shell. In order to set-up RtMidi,
+there are a few additional setup steps.
+
+* Setup the Pip and Python
+    * `apt-get remove upstart`
```

Comment:
Double backticks for inline code, please. The file is in ReST syntax, not markdown.